core/arch/arm/plat-stm/platform_config.h

```diff
@@ -282,13 +288,30 @@
 				CFG_SHMEM_SIZE)
 #endif
 
+#if defined(CFG_SECURE_DATA_PATH)
+#if !defined(CFG_TEE_SDP_MEM_BASE)
+/* default locate SDP memory right before the shared memory in DDR */
+#define CFG_TEE_SDP_TEST_MEM_SIZE	0x00300000
+
+#define CFG_TEE_SDP_MEM_SIZE	CFG_TEE_SDP_TEST_MEM_SIZE
+#define CFG_TEE_SDP_MEM_BASE	(CFG_DDR_TEETZ_RESERVED_START + \
+				CFG_DDR_TEETZ_RESERVED_SIZE - \
+				CFG_SHMEM_SIZE - \
+				CFG_TEE_SDP_MEM_SIZE)
+#endif
+#else
+#define CFG_TEE_SDP_TEST_MEM_SIZE	0
+#endif
+
 #if defined(CFG_WITH_PAGER)
 
 #define TZSRAM_BASE		CFG_CORE_TZSRAM_EMUL_START
 #define TZSRAM_SIZE		CFG_CORE_TZSRAM_EMUL_SIZE
 
 #define TZDRAM_BASE		CFG_DDR_TEETZ_RESERVED_START
-#define TZDRAM_SIZE		(CFG_DDR_TEETZ_RESERVED_SIZE - CFG_SHMEM_SIZE)
+#define TZDRAM_SIZE		(CFG_DDR_TEETZ_RESERVED_SIZE - \
+				CFG_SHMEM_SIZE - \
+				CFG_TEE_SDP_TEST_MEM_SIZE)
```

Comment:
`CFG_TEE_SDP_TEST_MEM_SIZE` is undefined here when both `CFG_SECURE_DATA_PATH` and `CFG_TEE_SDP_MEM_BASE` are defined, no?